data/en/reference/daemon/cli_options.yaml

```diff
@@ -0,0 +1,277 @@
+- category: |
+    `EC_PLUGIN`
+  evt_type: |
+    `pluginevent`
+  default: Yes
+  with_a: Yes
+  scap_file: Yes
+- category: |
+    `EC_SYSCALL`
+  evt_type: |
+    - `accept`
+    - `access`
+    - `bind`
+    - `bpf`
+    - `brk`
+    - `capset`
+    - `chdir`
+    - `chmod`
+    - `chown`
+    - `chroot`
+    - `clone`
+    - `clone3`
+    - `close`
+    - `connect`
+    - `copy_file_range`
+    - `creat`
+    - `dup`
+    - `dup2`
+    - `dup3`
+    - `epoll_create`
+    - `epoll_create1`
+    - `epoll_wait`
+    - `eventfd`
+    - `execve`
+    - `execveat`
+    - `fchdir`
+    - `fchmod`
+    - `fchmodat`
+    - `fchown`
+    - `fchownat`
+    - `fcntl`
+    - `flock`
+    - `fork`
+    - `fsconfig`
+    - `fstat`
+    - `fstat64`
+    - `futex`
+    - `getcwd`
+    - `getdents`
+    - `getdents64`
+    - `getegid`
+    - `geteuid`
+    - `getgid`
+    - `getpeername`
+    - `getresgid`
+    - `getresuid`
+    - `getrlimit`
+    - `getsockname`
+    - `getsockopt`
+    - `getuid`
+    - `inotify_init`
+    - `io_uring_enter`
+    - `io_uring_register`
+    - `io_uring_setup`
+    - `ioctl`
+    - `kill`
+    - `lchown`
+    - `link`
+    - `linkat`
+    - `listen`
+    - `llseek`
+    - `lseek`
+    - `lstat`
+    - `lstat64`
+    - `mkdir`
+    - `mkdirat`
+    - `mlock`
+    - `mlock2`
+    - `mlockall`
+    - `mmap`
+    - `mmap2`
+    - `mount`
+    - `mprotect`
+    - `munlock`
+    - `munlockall`
+    - `munmap`
+    - `nanosleep`
+    - `open`
+    - `open_by_handle_at`
+    - `openat`
+    - `openat2`
+    - `pipe`
+    - `poll`
+    - `ppoll`
+    - `pread`
+    - `preadv`
+    - `prlimit`
+    - `ptrace`
+    - `pwrite`
+    - `pwritev`
+    - `quotactl`
+    - `read`
+    - `readv`
+    - `recv`
+    - `recvfrom`
+    - `recvmmsg`
+    - `recvmsg`
+    - `rename`
+    - `renameat`
+    - `renameat2`
+    - `rmdir`
+    - `seccomp`
+    - `select`
+    - `semctl`
+    - `semget`
+    - `semop`
+    - `send`
+    - `sendfile`
+    - `sendmmsg`
+    - `sendmsg`
+    - `sendto`
+    - `setgid`
+    - `setns`
+    - `setpgid`
+    - `setresgid`
+    - `setresuid`
+    - `setrlimit`
+    - `setsid`
+    - `setsockopt`
+    - `setuid`
+    - `shutdown`
+    - `signalfd`
+    - `socket`
+    - `socketpair`
+    - `splice`
+    - `stat`
+    - `stat64`
+    - `symlink`
+    - `symlinkat`
+    - `syscall`
+    - `tgkill`
+    - `timerfd_create`
+    - `tkill`
+    - `umount`
+    - `unlink`
+    - `unlinkat`
+    - `unshare`
+    - `userfaultfd`
+    - `vfork`
+    - `write`
+    - `writev`
+  default: |
+    Only syscalls in *simple_set*:
```

Comment:
Note: we are changing this to the `sinsp_state_sc_set`, plus beware of the newly-introduced `base_syscalls` and the in-review `adaptive_base_syscalls` options. My suggestion is to wait for the adaptive syscall selection feature development to be over and then update this section. Reference:
- https://github.com/falcosecurity/falco/pull/2432
- https://github.com/falcosecurity/libs/pull/826
- https://github.com/falcosecurity/falco/issues/2433

Reply:
Makes sense! Thanks! 😄

Reply:
cc @incertum